pull-requests.md

```diff
@@ -1,63 +1,148 @@
+## Small
+A PR should be easy to understand, and address a single concern.
 
-Aim to deploy a pull request before submitting the next one. Frequently small deployments of functionality are easier to review and easier to understand in production.
+There should be a manageable number of changes in a PR. Try to make your PR as small as possible while still advancing the functionality of the product.
 
-* Don't mix multiple changes into the same pull request.
-* If you are doing bulk moves and deletions only have one kind of operation in a pull request
+If a PR is too complex, it is much easier for issues to be missed. In addition if trying to understand an aspect of the code change in the future, it becomes harder to understand if buried in a very large and complex PR.
+
+Aim to deploy a PR before submitting the next one. Frequently small deployments of functionality are easier to review and easier to understand in production.
 
 ## Releasable
+A PR should be releasable once its review is complete.
 
-A pull request should be releasable once its review is complete. The person submitting the pull request should be prepared to merge and release the change.
+The person submitting the PR should be prepared to merge and release the change.
 
-If these things are not true then you should address the blockers before raising the pull request.
+If these things are not true then you should address the blockers before raising the PR.
 
+>**Note**
+>Consider deploying the change to a pre-PROD environment to test your change in the wild.
 
-## The art of the pull request review
+## Descriptive
+A PR's description should include sufficient information on what the change is and why it is necessary.
 
-Good pull request reviews are extremely valuable and help share understanding of both the problem being addressed and its solution.
+This is an investment in your own future peace of mind. Once a PR is merged it will effectively be documentation of how the code has changed. There should be enough information on the reasoning behind the change to enable it to be understood by someone reading it for the first time.
 
-### A reviewer isn't responsible
+Where necessary, it should also describe how to test, what success looks like, and what a rollback would involve.
 
-When someone reviews a pull request they do not take responsibility for whether the code in the pull request is correct, whether it will work and that it will not have any side-effects elsewhere in the codebase.
+## Narrative
+It should be clear what the aim of each commit was.
 
-A review simply confirms that the pull request addresses its stated problem in a reasonable way and the solution has no obvious problems.
+A commit message should talk about _what_ changed, and _why_. Not _how_. How is the diff itself, and you don’t need to repeat it.
```

Comment:
Should we link to a good article on rewriting git history?

There's a good one from Github I cannot find and one @sndrs shared before I cannot remember the title of…

Reply:
Will merge this PR as is, but lets add the link once we find it.

Reply:
was it this? https://tekin.co.uk/2019/02/a-talk-about-revision-histories

it was @akash1810 that pointed me to it :)

Reply:
“It’s the circle of life… 🎶”